src/common/settings.js

```diff
@@ -599,6 +599,12 @@ const character_settings = {
         "description": "Unleash your divine soul to deal extra radiant damage equal to your level.",
         "type": "bool",
         "default": false
+    },
+    "charger-feat": {
+        "title": "Charger Feat - Extra Damage",
+        "description": "You charge into battle, lending weight to your blow!",
+        "type": "bool",
+        "default": false
```

Comment:
Seems to disable itself when used (makes sense as per the rules), I'd suggest adding a "(Apply to next roll)" like we have with sharpshooter.

Reply:
Done